Explorer/Assets/DCL_SkyBox_Vertex.hlsl

```diff
@@ -0,0 +1,114 @@
+#ifndef DCL_SKYBOX_VERTEX_INCLUDED
+#define DCL_SKYBOX_VERTEX_INCLUDED
+
+// Includes
+#include "Assets/DCL_SkyBox_Data.hlsl"
+
+// float4 _CurrentCubeFace;
+// #define _Current_CubeFace _CurrentCubeFace.x
+// static const int _CubeFaceSTUFF = 5;
+// #define _Current_CubeFace _CubeFaceSTUFF
+
+
+float3 ComputeCubeDirection(float2 globalTexcoord)
+{
+    float2 xy = (globalTexcoord * 2.0) - 1.0;
+    //return float3(-xy.x, -xy.y, -1.0);
+    //float3 direction = float3(-xy.x, -xy.y, -1.0);
+
+    // if(_Current_CubeFace == 0.0f)
```

Comment:
Maybe comment here as to why it is commented out, explaining the unity support ticket that is pending

Reply:
Agreed